hw/lint/Makefile

```diff
@@ -0,0 +1,103 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Makefile with ascentlint and verilator-lint targets for OpenTitan
+#
+# TODO: currently we cannot support parallel builds since some fusesoc cores
+# define filesets with files outside the current folder (e.g. using relative
+# path prefixes such as "../../"). this can cause collisions between parallel
+# builds since they are not nicely contained within the same folder. This should
+# be solved by reworking the fusesoc core files (especially the top-level one).
+
+CORE_ROOT ?= ../../
+REPORT_DIR ?= reports
+
+IPS ?=  ip-aes                 \
+        ip-alert_handler       \
+        ip-flash_ctrl          \
+        ip-gpio                \
+        ip-hmac                \
+        ip-i2c                 \
+        ip-nmi_gen             \
+        ip-padctrl             \
+        ip-padring             \
+        ip-pinmux              \
+        ip-rv_core_ibex        \
+        ip-rv_dm               \
+        ip-rv_plic_example     \
+        ip-rv_timer            \
+        ip-spi_device          \
+        ip-uart                \
+        ip-usbdev              \
+        ip-usb_fs_nb_pe        \
+        ip-usbuart             \
+        tlul-socket_1n         \
+        tlul-socket_m1         \
+        tlul-adapter_reg       \
+        tlul-adapter_sram      \
+        tlul-sram2tlul         \
+        systems-top_earlgrey
+
+ips_lint  = $(addsuffix _lint, $(IPS))
+ips_vlint = $(addsuffix _vlint, $(IPS))
+
+######################
+# ascentlint targets #
+######################
+
+# lint all discovered targets and make a report
+all: lint
+	$(MAKE) report
+
+lint: clean
+	@echo Discovered lint targets:
+	@echo -e "\n $(patsubst %,%\\n,$(strip $(ips_lint)))"
+	$(MAKE) $(ips_lint)
+
+$(ips_lint):
+	\rm -rf build
```

Comment:
Please remove the backslash from all `rm` calls.

Reply:
ah yeah that should not be there. will remove.

Reply:
done